setup.cfg

```diff
@@ -0,0 +1,4 @@
+[nosetests]
+tests = test/unit,
+        utils,
+        roles/openshift_master_facts/test,
```

Comment:
should `roles/openshift_master_facts/test/*` be moved to `roles/openshift_master_facts/test/unit/` as well?